server/odc-service/src/main/java/com/oceanbase/odc/service/task/config/TaskFrameworkProperties.java

```diff
@@ -42,6 +42,21 @@ public class TaskFrameworkProperties {
     @NestedConfigurationProperty
     private K8sProperties k8s = new K8sProperties();
 
+    // job will be expired after this duration
+    private int jobExpiredDurationSeconds = 10 * 60;
```

Comment:
It is best if the framework parameters can be configured dynamically

Reply:
this framework parameters will be set in meta-db at later pr